addons/escoria-ui-9verbs/game.gd

```diff
@@ -224,9 +224,31 @@ func pause_game():
 		escoria.main.current_scene.hide()
 		escoria.set_game_paused(true)
 
+
 func _on_MenuButton_pressed() -> void:
 	pause_game()
 	
 	
 func _on_action_finished() -> void:
 	verbs_menu.unselect_actions()
+
+
+func show_crash_popup(files = []) -> void:
+	connect("crash_popup_confirmed", escoria, "quit", 
+		[], CONNECT_ONESHOT)
+	var crash_popup = escoria.resource_cache.get_resource(
+		"res://addons/escoria-core/ui_library/crash_popup/crash_popup.tscn"
+	).instance()
+	add_child(crash_popup)
+	move_child(crash_popup, 0)
```

Comment:
Why is this line required?

Reply:
Mistake. The idea was originally to ensure the popup is drawn over everything, but this was not correct anyway. Thanks!